tests/test_main.py

```diff
@@ -19,26 +19,27 @@ def test_car():
         ([Car(2, 1, "Ford")], CarWashStation(3, 9, 4.2, 11), 22.4),
         ([Car(2, 9, "Ford")], CarWashStation(3, 8, 4.2, 11), 0),
         (
-            [Car(3, 3, "BMW"), Car(4, 5, "Audi"), Car(7, 1, "Mercedes")],
-            CarWashStation(6, 7, 3.9, 11),
-            40.3,
+                [Car(3, 3, "BMW"), Car(4, 5, "Audi"), Car(7, 1, "Mercedes")],
+                CarWashStation(6, 7, 3.9, 11),
+                40.3,
         ),
         (
-            [Car(3, 3, "BMW"), Car(4, 5, "Audi"), Car(7, 9, "Mercedes")],
-            CarWashStation(6, 7, 3.9, 11),
-            13.0,
+                [Car(3, 3, "BMW"), Car(4, 5, "Audi"), Car(7, 9, "Mercedes")],
+                CarWashStation(6, 7, 3.9, 11),
+                13.0,
         ),
         (
-            [Car(3, 8, "BMW"), Car(4, 8, "Audi"), Car(7, 9, "Mercedes")],
-            CarWashStation(6, 7, 3.9, 11),
-            0,
+                [Car(3, 8, "BMW"), Car(4, 8, "Audi"), Car(7, 9, "Mercedes")],
+                CarWashStation(6, 7, 3.9, 11),
+                0,
         ),
```

Comment:
please don't change tests

Reply:
Sorry, there was a whitespace issue in the tests, and Black auto-fixed it. I included tests for the next tasks.